app/controllers/users_controller.rb

```diff
@@ -6,6 +6,14 @@ class UsersController < ApplicationController
   def index
     @q = User.ransack(params[:q])
     @users = policy_scope @q.result(distinct: true).includes(:outreach_event_types, :cohort)
+
+    respond_to do |format|
+      format.html
+      format.csv { 
+        authorize @users, :to_csv?
+        send_data User.to_csv(@users), filename: "mrv-users.csv"
+      }
+    end
```

Comment:
This is such a clever idea to reuse the method! I really like that my search results are also persisted when I export to CSV :D